src/Form/FormAutosuggest.jsx

```diff
@@ -157,6 +157,15 @@ function FormAutosuggest({
     };
   });
 
+  useEffect(() => {
+    if (value !== state.displayValue) {
+      setState(prevState => ({
+        ...prevState,
+        displayValue: value,
+      }));
+    }
+  }, [value]);
```

Comment:
I'm not sure why the CI linter is passing here, but locally ESLint complains that `state.displayValue` is missing from the `useEffect` dependency array. Without it, if/when `state.displayValue` changes, this effect won't trigger.

<img width="584" alt="image" src="https://github.com/openedx/paragon/assets/2828721/67b10a42-6f62-4ac9-8abb-6d51cf26c96a">